my-app/src/components/content/content.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import './content.css';
+
+const Content = () => {
+
+    return (
```

Comment:
[global]: redundant return. Make it shorter: const Content = () => (
                                                          <div className=...